### ISSUE TITLE
Commit 9af8d6912a42e95813943258df86a3edc4f2f1b0 breaks GOOS=windows

### ISSUE THREAD

**Issue by cwegener** [influxdata/telegraf#838]
Commit 9af8d6912a42e95813943258df86a3edc4f2f1b0 breaks GOOS=windows with the following error:

```
# github.com/influxdata/telegraf/internal/config
internal/config/config.go:338: cannot use subTable (type *"github.com/influxdata/toml/ast".Table) as type *"github.com/naoina/toml/ast".Table in argument to config.UnmarshalTable
internal/config/config.go:343: cannot use subTable (type *"github.com/influxdata/toml/ast".Table) as type *"github.com/naoina/toml/ast".Table in argument to config.UnmarshalTable
internal/config/config.go:420: cannot use table (type *"github.com/influxdata/toml/ast".Table) as type *"github.com/naoina/toml/ast".Table in argument to config.UnmarshalTable
internal/config/config.go:464: cannot use table (type *"github.com/influxdata/toml/ast".Table) as type *"github.com/naoina/toml/ast".Table in argument to config.UnmarshalTable
internal/config/config.go:636: cannot use subtbl (type *"github.com/influxdata/toml/ast".Table) as type *"github.com/naoina/toml/ast".Table in argument to config.UnmarshalTable
```

Reverting 9af8d6912a42e95813943258df86a3edc4f2f1b0 works around the error.

I can see that influxdata/toml is missing from Godeps_windows and inflxudata/config has a different version in  9af8d6912a42e95813943258df86a3edc4f2f1b0
That probably has something to do with the error....

**Comment by cwegener** [comment]
This issue can be fixed by resolving issue #839

**Comment by sparrc** [comment]
closing as dupe of #839
